packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3553,12 +3654,14 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
         ),
       ),
     );
+
+    return child;
   }
 
   /// Builds [TextSpan] from current editing value.
   ///
   /// By default makes text in composing range appear as underlined.
-  /// Descendants can override this method to customize appearance of text.
+  /// Descendants can override this method to customize appearance of text
```

Comment:
Nit: Comment should end with a `.`

Reply:
Not sure how I ended up removing that :grin: